plugins/destination/bigquery/client/client.go

```diff
@@ -16,6 +16,7 @@ type Client struct {
 	logger     zerolog.Logger
 	spec       specs.Destination
 	metrics    plugins.DestinationMetrics
+	batchSize  int
```

Comment:
I see only initialization of this field.
Where is the propagation to the write?

Reply:
https://github.com/cloudquery/cloudquery/pull/5681